README.md

```diff
@@ -387,6 +387,16 @@ listed in alphabetical order by their surname.
 - Race Mahoney
 - Nicholas Tocci
 
+## Presentations
+
+GatorGrader's creators give presentations about the development, use, and
+assessment of the tool. Please contact one of the developers if you would like
+to feature a presentation about GatorGrader at your technical conference. The
+following list includes some recent of our presentations:
```

Comment:
Probably should be reworded, perhaps to "The following list includes some of our recent presentations"

Reply:
Thanks for pointing this out. I fixed it!